### PR TITLE
8288528: broken links in java.desktop

### PR DESCRIPTION
**Pull request by prsadhuk**
doccheck reports some broken links which are rectified

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8288528](https://bugs.openjdk.org/browse/JDK-8288528): broken links in java.desktop

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**) ⚠️ Review applies to [83910eb0](https://git.openjdk.org/jdk/pull/9195/files/83910eb00a989deb98110e59b047cdd373c32de2)
 * @kristylee88 (no known github.com user name / role) ⚠️ Review applies to [83910eb0](https://git.openjdk.org/jdk/pull/9195/files/83910eb00a989deb98110e59b047cdd373c32de2)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/9195/head:pull/9195` \
`$ git checkout pull/9195`

Update a local copy of the PR: \
`$ git checkout pull/9195` \
`$ git pull https://git.openjdk.org/jdk pull/9195/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 9195`

View PR using the GUI difftool: \
`$ git pr show -t 9195`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/9195.diff">https://git.openjdk.org/jdk/pull/9195.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h3>
 Prior to 1.4, the helper threads were never terminated.
 <p>
 Starting with 1.4, the behavior has changed as a result of the fix for
-<a href="https://bugs.java.com/view_bug.do?bug_id=4030718">
+<a href="https://bugs.openjdk.org/browse/JDK-4030718">
```
Comment (aivanov-jdk) [inline src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html:124]: The old URL has changed to https://bugs.java.com/bugdatabase/view_bug.do?bug_id=4030718

I am for using JBS links though.

File: src/java.desktop/share/classes/javax/swing/plaf/metal/MetalLookAndFeel.java [hunk 2]
```diff
@@ -2139,9 +2139,8 @@ public Icon getDisabledSelectedIcon(JComponent component, Icon icon) {
 
 
     /**
-     * Returns a {@code LayoutStyle} implementing the Java look and feel
-     * design guidelines as specified at
-     * <a href="http://www.oracle.com/technetwork/java/hig-136467.html">http://www.oracle.com/technetwork/java/hig-136467.html</a>.
+     * Returns a {@code LayoutStyle} implementing Java look and feel
+     * design guidelines.
```
Comment (aivanov-jdk) [inline src/java.desktop/share/classes/javax/swing/plaf/metal/MetalLookAndFeel.java:2143]:
```suggestion
     * Returns a {@code LayoutStyle} implementing the Java look and feel
     * design guidelines.
```

**Review by aivanov-jdk — APPROVED** [review]

File: src/java.desktop/share/classes/javax/swing/plaf/multi/doc-files/multi_tsc.html [hunk 3]
```diff
@@ -39,7 +39,7 @@ <h1>Using the Multiplexing Look and Feel</h1>
 <i>
 This document is based on an article
 originally published in
-<a href="http://www.oracle.com/technetwork/java/javase/tech/articles-jsp-139072.html"
+<a href="https://www.comp.nus.edu.sg/~cs3283/ftp/Java/swingConnect/archive/plaf_papers_arch/multi_update/multi_update.html"
```
Comment (prrace) [inline src/java.desktop/share/classes/javax/swing/plaf/multi/doc-files/multi_tsc.html:42]: Well I am 100% sure this article wasn't originally published by the national university of singapore.
And we definitely should NOT be linking there. 
I actually think for most of these we'd be better off getting rid of the link entirely.

**Review by prrace — CHANGES_REQUESTED** [review]

**Review by kristylee88 — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

File: src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html [hunk 4]
```diff
@@ -154,11 +154,11 @@ <h3>
 <ul>
   <li> Other packages can create displayable components for internal
        needs and never make them undisplayable. See
-<a href="https://bugs.java.com/view_bug.do?bug_id=4515058">
+<a href="https://bugs.openjdk.org/browse/JDK-4515058">
```
Comment (prrace) [inline src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html:157]: This is about Solaris IMF .. I think we don't need to worry about it any more.

File: src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html [hunk 5]
```diff
@@ -154,11 +154,11 @@ <h3>
...
 4515058</a>,
-<a href="https://bugs.java.com/view_bug.do?bug_id=4671025">
+<a href="https://bugs.openjdk.org/browse/JDK-4671025">
```
Comment (prrace) [inline src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html:159]: And even better it was resolved in 1.5 under this bug id which I guess was still open when this doc was written

File: src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html [hunk 6]
```diff
@@ -154,11 +154,11 @@ <h3>
...
 4671025</a>, and
-<a href="https://bugs.java.com/view_bug.do?bug_id=4465537">
+<a href="https://bugs.openjdk.org/browse/JDK-4465537">
```
Comment (prrace) [inline src/java.desktop/share/classes/java/awt/doc-files/AWTThreadIssues.html:161]: This bug was about some off-screen image problem with JScrollBar
It was closed as a dup of 4417287 which was fixed in 1.4

So in summary it is wrong to keep these linked bugs in here.
Remove the lot of them

**Review by prrace — CHANGES_REQUESTED** [review]

**Comment by prsadhuk** [comment]
will be done in jdk19 forked repo

**Pull request closed without merging**